docs/source/theming/custom-styling.md

```diff
@@ -7,6 +7,11 @@ myst:
     "keywords": "Volto, Plone, frontend, React, Custom, Styling, Semantic UI, LESS, Pastanaga"
 ---
 
+```{deprecated} Volto 18
```

Comment:
Move all admonitions under a heading, either the title or appropriate subheading. Also use an include for all of these copy-pasta's instead.

Reply:
Done.